meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report

```diff
@@ -113,7 +113,14 @@ printf '########################################                            \r'
 decode-dimms > logs/decode-dimms.log 2> logs/decode-dimms.err.log
 printf '##########################################                          \r'
 
+# Dump cbmem table if Dasharo is installed
+if [ -n "`fwupdtool hwids | grep -i dasharo`" ]
```

Comment:
@kotylamichal please take a look at issue description. Every coreboot based fw should be able to dump cbmem table. So maybe we could just run cbmem dump, if there will be nothing the to report we probable will get some error or so which is ok to save in log.

Reply:
@TomaszAIR done